build/grunt.js

```diff
@@ -398,9 +406,10 @@ module.exports = function(grunt) {
     'shell:lint',
     'clean:build',
 
-    'shell:rollupall',
+    // 'shell:rollupall',
```

Comment:
this probably shouldn't be commented out.

Reply:
woops yeah, I was testing with it commented out.